explorer/src/lib/components/block-details/BlockDetails.css

```diff
@@ -0,0 +1,66 @@
+.block-details {
+  display: flex;
+  flex-direction: column;
+  gap: 1.25rem;
+}
+
+.block-details__header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.block-details__header-heading {
+  font-size: 1.5rem;
+  font-weight: 400;
+}
+
+.block-details__list {
+  display: inline-grid;
+  grid-template-columns: 1fr 3fr;
+  font-size: 0.875rem;
+  column-gap: 1.875rem;
+}
+
+.block-details__list-timestamp {
+  text-transform: uppercase;
+}
+
+.block-details__list-anchor {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0.0625rem;
+  color: var(--primary-color);
+}
+
+.block-details__list-term-gas-used {
+  width: 12.5rem;
+}
+
+.block-details__list-definition-card {
+  padding: 0.625rem 0.875rem;
+  line-height: 1.3125rem;
+  width: 100%;
+}
+
+@media (max-width: 640px) {
```

Comment:
This doesn't follow the mobile-first approach.

Reply:
How would you have written it?